pkg/driver/node/mounter/credential_provider.go

```diff
@@ -96,7 +89,7 @@ func (c *CredentialProvider) Provide(ctx context.Context, volumeID string, volum
 		return nil, status.Error(codes.InvalidArgument, "Missing volume context")
 	}
 
-	authenticationSource := volumeContext[VolumeCtxAuthenticationSource]
+	authenticationSource := volumeContext[volumecontext.AuthenticationSource]
```

Comment:
Not blocking: The only difference here being capitalisation makes this more challenging to read at first glance.

Reply:
I can rename variables to `volumeCtx`, to have similar naming convention to built-in `context` package and `ctx` variable used widely.

Reply:
Addressed with https://github.com/awslabs/mountpoint-s3-csi-driver/pull/340/commits/5da88fc2fe558f98706c7f8092be45cd35b6e511